mlir/test/Integration/Dialect/Vector/CPU/ArmSVE/test-scalable-deinterleave.mlir

```diff
@@ -1,3 +1,4 @@
+// REQUIRES: arm-emulator-executable
```

Comment:
[nit] Leave an empty line between `REQUIRES` and `DEFINE` to clearly separate the pre-conditions and the test itself.